babel-env-vars.js

```diff
@@ -0,0 +1,18 @@
+
+const pkg = require('./package.json')
+const fs = require('fs')
+
+const VERSION = fs.readFileSync('./VERSION', 'utf-8')
+
```

Comment:
This file is used to inject a "version" into the bundle.  This is complicated right now by the fact that NPM would build with package.json.version (0.0.9) and CDN will build with VERSION file's value (1222), as well as subversion attributes such as PROD, DEV, LOCAL, etc.. Once aligned, this process will get much simpler.